tests/pytests/functional/fileserver/test_roots.py

```diff
@@ -9,8 +14,12 @@
 
 @pytest.fixture(scope="module")
 def configure_loader_modules(base_env_state_tree_root_dir):
+    cachedir = os.path.join(RUNTIME_VARS.TMP, "__salt_test_fileserver_roots_cache_dir")
+    if not os.path.isdir(cachedir):
+        os.makedirs(cachedir)
     opts = salt.config.DEFAULT_MINION_OPTS.copy()
-    print(base_env_state_tree_root_dir)
+    print(base_env_state_tree_root_dir, end=" ")
```

Comment:
Is there any purpose to this `print`?  I wonder if it was leftover from versions prior that just never got caught.

Reply:
Yeah, let's nuke it